gsi_wdl_tools/workflow_info.py

```diff
@@ -63,13 +64,29 @@ def outputs(self):
                 if val is not None:
                     description = val
                 else:
-                    output_file_name = str(output.info.expr)
-                    description = output_descriptions.get(output_file_name)
+                    raise Exception(f"output description is missing for {name}")
             else:
-                raise Exception('Unsupported input type')
-            if not description:
-                raise Exception(f"output description is missing for {name}")
-            outputs.append(Output(name=name, wdl_type=wdl_type, description=description))
+                raise Exception('Unsupported input type')             
+                
+            vidarr_label = [] 
+                       
+            # Check if 'vidarr_label' exists in the output description
+            if 'vidarr_label' in output_descriptions[output.name]:
+                vidarr_label.append(('vidarr_label', output_descriptions[output.name]['vidarr_label']))
+                # If vidarr_label exists, should convert file to file-with-labels
+                if wdl_type == "File":
```

Comment:
I'm not sure if we should be changing the type in the readme, as if the wdl file in the repo were to be executed as is, it would be of type File. We only do File -> Pair[File,Map[String,String]] if installed in Vidarr.

Reply:
I think it would be confusing to show a File with a label since File and Pair[File, Map[String, String]] map to two different vidarr types. I'm less familiar with this repo - what are these readmes used for? Would `File (will upgrade to file-with-labels)` be a good compromise?

Reply:
This tool takes a wdl file and generates the readme's present in wdl workflow repos (e.g. https://github.com/oicr-gsi/bwaMem/?tab=readme-ov-file#outputs). The fact we're changing the type is more of a Vidarr operational detail. Running the workflow outside of Vidarr will still produce output of type File.

Reply:
I understand that, but this is specifically a gsi tool and Files cannot have labels in vidarr. Some sort of indication is probably warranted.